enhancements/proposals/1665-metadata/README.md

```diff
@@ -0,0 +1,122 @@
+# Support for ScyllaDB Alternator
+
+## Summary
+
+This proposal aims to introduce basic support for handling metadata in managed objects by ScyllaClusters.
+It aims to make minimal changes to satisfy the use cases presented so far.
+
+## Motivation
+
+ScyllaCluster users need to specify labels and annotations for various reasons including external integrations or object discovery / tracking.
+
+### Goals
+
+- Inherit ScyllaCluster labels and annotation into managed objects (aka default labels)
+- Allow specifying distinct labels and annotations for node services
+- Allow specifying distinct labels and annotations for pods
+- Allow specifying distinct labels and annotations for ingresses
+
+### Non-Goals
+
+- Allow unique metadata for every single object we create 
+
+## Proposal
+
+I'd like to use the existing ScyllaCluster.metadata (labels and annotations) and inherit them in every object we create,
+unless there is an override.
+In the case when using an override, the ScyllaCluster.metadata won't be used.
```

Comment:
Wouldn't it be more intuitive to merge the two? With the priority being managed>object-defined>cluster-defined. I can imagine wanting to have some labels/annotations applied to all objects and then a subset applied to say services only.

Reply:
you wouldn't be able to remove an annotation used on the top level object